tests/unit/test-cmd/test.build.js

```diff
@@ -1,6 +1,7 @@
 import path from 'path';
+import fs from 'fs/promises';
+import { writeFileSync } from 'fs';
```

Comment:
In some cases I used the single export instead. This pattern was also already used in parts of the repo. However this is not always possible due to stubbing.

Reply:
Makes sense (and thanks a lot for mentioning the motivations behind these choices explicitly in this comment, that really helped to speed up the review and sign off :heart:)